app/assets/stylesheets/custom.css.scss

```diff
@@ -139,7 +139,7 @@ h1 {
   padding:4px;
   border-radius: 100px;
   color: #fff8dc;
-  background-color: #ff0000;
+  background-color: red;
```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.